docs/setup-cluster/deploy-cluster/k8s/custom-pod-specs.rst

```diff
@@ -60,14 +63,37 @@ Determined does not currently support configuring:
 
 -  Restart Policy - This is always set to ``Never``.
 
-As part of their pod spec, users can specify ``initContainers`` and ``containers``. Additionally
-users can configure the ``determined-container`` that executes the task (e.g., training), by setting
-the container name in the pod spec to ``determined-container``. For the ``determined-container``,
-Determined currently supports configuring:
+Supported
+=========
+
+As part of your pod spec, you can specify ``initContainers`` and ``containers``. Additionally you
+can configure the ``determined-container`` that executes the task (e.g., training), by setting the
+container name in the pod spec to ``determined-container``. For the ``determined-container``,
+
+Determined supports configuring the following fields:
 
 -  Resource requests and limits (except GPU resources).
+
 -  Volume mounts and volumes.
 
+-  All ``securityContext`` fields within the pod spec of the ``determined-container`` container
+   except for ``RunAsUser`` and ``RunAsGroup``. For those fields, use ``det user
+   link-with-agent-user`` instead.
```

Comment:
Nit: do you mean to add a line break  user and  link-with-agent-user?
<img width="1004" alt="image" src="https://github.com/determined-ai/determined/assets/124635598/9c477f6b-dce4-4428-9e1e-ff54ad3005bf">

Reply:
fixed

Reply:
really fixed this time :-;